index.rst

```diff
@@ -295,6 +295,7 @@ Cookbook
     Mirabella Genio Bulb, cookbook/mirabella-genio-bulb, cookbook-mirabella-genio-b22-rgbw.jpg
     Garage Door, cookbook/garage-door, window-open.svg
     Brilliant / Mirabella Genio Smart Plugs, cookbook/brilliant-mirabella-genio-smart-plugs, cookbook-brilliant-mirabella-genio-smart-plugs.jpg
+    SSD1306 OLED Guide, cookbook/oled, oled.png
```

Comment:
This should not be a PNG (png uses way too much space for these types of images). Make it a JPG.